README.md

```diff
@@ -1,26 +1,33 @@
 # OTel Me If It's Right
 
-Checker if the implementation of OpenTelemetry instrumentation is correct.
+Checker for if the implementation of OpenTelemetry instrumentation is correct by scanning the code in your repository.
```

Comment:
I realize that this is not all - we also try to make a call to the endpoint using your api token: https://github.com/grafana/otel-checker/blob/main/checks/grafana/grafanaChecker.go#L113